internal/compiler/tests/syntax/lookup/property.slint

```diff
@@ -41,10 +41,15 @@ export X := Rectangle {
 //              ^error{Element 'Comp' does not have a property 'not_exist'}
     }
 
+    img := Image{}
+
     callback dummy;
     dummy => {
-        SlintInternal.dark-color-scheme = false;
```

Comment:
SlintInternal now would give another error, so i found other builtin fake properties that give the same error.